learning/tour-of-beam/frontend/README.md

```diff
@@ -17,47 +17,67 @@
      under the License.
  -->
 
- # Tour of Beam
- These are the main sources of the Tour of Beam website.
+# Tour of Beam
 
- # About
+These are the main sources of the Tour of Beam website.
+
+# About
 
 ## Getting started
-Running, debugging, and testing all require this first step that fetches
-dependencies and generates code:
+
+This project relies on generated code for some functionality:
+deserializers, test mocks, constants for asset files,
+extracted Beam symbols for the editor, etc.
+
+All generated files are version-controlled, so after checkout the project is immediately runnable.
+However, after changes you may need to re-run code generation:
 
 ```bash
-cd ../../../playground/frontend/playground_components
-flutter pub get
-flutter pub run build_runner build
-cd ../../../learning/tour-of-beam/frontend
-flutter pub get
+cd beam
+./gradlew :playground:frontend:playground_components:generateCode
+cd learning/tour-of-beam/frontend
 flutter pub run build_runner build
```

Comment:
Additionally, why is there a need for `flutter pub run build_runner build` when we see files such as `learning/tour-of-beam/frontend/lib/auth/method.g.dart`?

Reply:
Just above it reads
> However, after changes you may need to re-run code generation: